### PR TITLE
Lookup pretty

### PR DESCRIPTION
**Pull request by dignifiedquire**
- Fix linting
- Update devDependencies
- Add `lookupPretty`
- Improve docs
- Improve code organization

Ref https://github.com/ipfs/node-ipfsd-ctl/issues/18

### CODE REVIEW

File: lib/format.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  if (data[1]) obj.country_code = data[1]
+  if (data[2]) obj.region_code = data[2]
+  if (data[3]) obj.city = data[3]
+  if (data[4]) obj.postal_code = data[4]
+  if (data[5]) obj.latitude = parseFloat(data[5])
+  if (data[6]) obj.longitude = parseFloat(data[6])
+  if (data[7]) obj.metro_code = data[7]
+  if (data[8]) obj.area_code = data[8]
+
+  obj.planet = PLANET
```
Comment (jbenet) [inline lib/format.js:19]: :+1:

Comment (jbenet) [inline lib/format.js:19]: :)

File: package.json [hunk 2]
```diff
@@ -33,7 +34,7 @@
   },
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "git diff --name-only --cached --relative | egrep .js$ | xargs --no-run-if-empty standard"
+    "lint": "node_modules/.bin/standard"
```
Comment (jbenet) [inline package.json:37]: i think @diasdavid used the `git diff` mode? not sure what the benefit is there

**Comment by jbenet** [comment]
this LGTM, though i haven't checked the js code

File: package.json (hunk 2 above)
Comment (daviddias) [inline package.json:37]: nah, that `gif diff` madness is not mine :) and I already found times that super fast linting check (with `git diff`) didn't caught linting errors that lead to mem leaks in another project. I recommend running standard and let it check everything.

**Comment by krl** [comment]
Looking good!

**Comment by dignifiedquire** [comment]
@krl feel free to merge when you like it

**Comment by daviddias** [comment]
Merging it :)

**Comment by daviddias** [comment]
@jbenet @krl when you can, can you publish it to npm? (or give me perm, happy to do it)

**Comment by jbenet** [comment]
@diasdavid should be owner now.

**Comment by daviddias** [comment]
thanks @jbenet 

@Dignifiedquire published https://www.npmjs.com/package/ipfs-geoip :)

**Comment by dignifiedquire** [comment]
thanks

**Pull request merged**